invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/deposit/RDMDepositForm.js

```diff
@@ -106,388 +107,448 @@ export class RDMDepositForm extends Component {
+                    "metadata.publication_date",
+                    "metadata.creators",
+                    "metadata.description",
+                    "metadata.additional_descriptions",
+                    "metadata.rights",
+                  ]}
+                  active
+                  label={i18next.t("Basic information")}
                 >
-                  <ResourceTypeField
-                    options={this.vocabularies.metadata.resource_type}
+                  <Overridable
+                    id="InvenioAppRdm.Deposit.PIDField.container"
+                    pids={this.config.pids}
+                    record={record}
+                  >
+                    <Fragment>
+                      {this.config.pids.map((pid) => (
+                        <Fragment key={pid.scheme}>
+                          <PIDField
+                            btnLabelDiscardPID={pid.btn_label_discard_pid}
+                            btnLabelGetPID={pid.btn_label_get_pid}
+                            canBeManaged={pid.can_be_managed}
+                            canBeUnmanaged={pid.can_be_unmanaged}
+                            fieldPath={`pids.${pid.scheme}`}
+                            fieldLabel={pid.field_label}
+                            isEditingPublishedRecord={
+                              record.is_published === true // is_published is `null` at first upload
+                            }
+                            managedHelpText={pid.managed_help_text}
+                            pidLabel={pid.pid_label}
+                            pidPlaceholder={pid.pid_placeholder}
+                            pidType={pid.scheme}
+                            unmanagedHelpText={pid.unmanaged_help_text}
+                            required
+                          />
+                        </Fragment>
+                      ))}
+                    </Fragment>
+                  </Overridable>
+
+                  <Overridable
+                    id="InvenioAppRdm.Deposit.ResourceTypeField.container"
+                    vocabularies={this.vocabularies}
                     fieldPath="metadata.resource_type"
-                    required
-                  />
-                </Overridable>
+                  >
+                    <ResourceTypeField
+                      options={this.vocabularies.metadata.resource_type}
+                      fieldPath="metadata.resource_type"
+                      required
+                    />
+                  </Overridable>
 
-                <Overridable
-                  id="InvenioAppRdm.Deposit.TitlesField.layout"
-                  vocabularies={this.vocabularies}
-                >
-                  <TitlesField
-                    options={this.vocabularies.metadata.titles}
+                  <Overridable
+                    id="InvenioAppRdm.Deposit.TitlesField.container"
+                    vocabularies={this.vocabularies}
```

Comment:
Here it's good to pass the `record={record}` so that you can override the `TitlesField` component